src/routes/Parameters.svelte

```diff
@@ -42,6 +58,15 @@
 					<input type="number" bind:value={params.factorTransfer} class="border-0 w-16" />
 				</td>
 			</tr>
+			<tr>
+				<td colspan="2" class="px-2 first:pl-5 last:pr-5 py-3 whitespace-nowrap decoration-pink-500"
+					><label for="json-io">JSON Import/Export</label><br /><textarea
```

Comment:
formatting

Reply:
Is it better now? Formatting with Prettier does some weird looking stuff. I did some manual reformatting.

Reply:
Yes. Seems like this has to be done manually. Or we could configure it somehow somewhere. But for now, it's a manual task.